twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,12 +73,23 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isAccessible = (component: IPaneComponent) => {
+        const isLastPaneOne = component.pane === paneType.ONE && paneComponents.filter(x => x.pane === paneType.ONE).pop()?.key === component.key;
+        const isLastPaneTwo = component.pane === paneType.TWO && paneComponents.filter(x => x.pane === paneType.TWO).pop()?.key === component.key;
+
+        if ((paneRects.length > 1 && (isLastPaneOne || isLastPaneTwo && !isPaneOneExtended())) || (paneRects.length === 1 && isLastPaneOne)) {
```

Comment:
`(paneRects.length > 1 && (isLastPaneOne || isLastPaneTwo && !isPaneOneExtended()))`
re. ^ if isLastPaneOne == true, does it not matter then is it's extended?

can add a brief comment explaining this logic